File: TG001/Models/Migracao/ContasMigracaoMigradas.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace TG001.Models.Migracao
{
    public class ContasMigracaoMigradas
    {
        [Key]
        public int ID { get; set; }

        [ForeignKey("ContasMigracao")]
        public int ContasMigracaoID { get; set; }

        [Required]
        public int LoteID { get; set; }

        [Required]
        public DateTime DataMigracao { get; set; }

        public virtual ContasMigracao ContasMigracao { get; set; }
    }
}

File: TG001/ClientApp/src/app/Classes/Contas/Migracao/ContasMigracaoMigradas.ts
import { ContasMigracao } from "./ContasMigracao"

export class ContasMigracaoMigradas {
	constructor(

		public ID: number,
		public ContasMigracaoID: number,
		public LoteID: number,
		public DataMigracao: Date,
		public ContaMigracao: ContasMigracao,
	) { }
}

